go_api_server/database/util.go

```
@@ -1,23 +1,103 @@
+type SelectQuery struct {
+	Table           string
+	Fields          []string
+	LogicExpression LogicExpression
+	PagingInfo      PagingInfo
+}
+
+// Determines how many rows to return and what index to start
+//   - startIndex Offset for sql query
+//   - batchSize Limit for sql queru
+type PagingInfo struct {
+	startIndex int
```

golangci-lint (1.64.8)

field `startIndex` is unused

(unused)